gtsfm/loader/loader_base.py

```diff
@@ -198,7 +200,10 @@ def get_camera_intrinsics(self, index: int) -> Optional[gtsfm_types.CALIBRATION_
     def get_image_shape(self, idx: int) -> Tuple[int, int]:
         """Return a (H,W) tuple for each image"""
         image = self.get_image(idx)
-        return (image.height, image.width)
+        if image is not None:
+            return (image.height, image.width)
+        else:
+            return (0, 0)
```

Comment:
oh yeah?

Reply:
This was a hack we did for Hilti as there was one image which was not present. I'll remove it and throw an error